src/main/java/org/eclipse/basyx/aas/factory/aasx/AASXToMetamodelConverter.java

```diff
@@ -297,6 +297,15 @@ public void unzipRelatedFiles() throws IOException, ParserConfigurationException
 		unzipRelatedFiles(getTemporaryDirPath());
 	}
 	
+	/**
+	 * Unzips all files referenced by the aasx file according to its relationships
+	 * and a relative child path
+	 */
+	public void unzipRelatedFilesToChildPath(String childPath) throws IOException, ParserConfigurationException, SAXException, URISyntaxException, InvalidFormatException {
```

Comment:
Still missing a test case for this function in the ``TestAASXToMetamodelConverterFromBaSyx``-Class.